google_chat_ros/action/GoogleChatREST.action

```diff
@@ -0,0 +1,10 @@
+# Define the goal
+string space
+string message_type
+string content # if message_type is text, string, else json.
```

Comment:
According to your README.md, should `message_type` be "text" or "card"?
Please comment about `message_type` options.